core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -524,6 +525,18 @@ public T maxTraceEvents(int maxTraceEvents) {
     throw new UnsupportedOperationException();
   }
 
+  /**
+   * Sets the proxy detector to be used in addresses name resolution. If <code>null</code> is passed
+   * default proxy detector {@link GrpcUtil#getDefaultProxyDetector()} will be used
```

Comment:
nit: just "default proxy detector". The GrpcUtil thing is internal, and it isn't really pertinent, so let's not mention it. If you _really_ wanted, you could mention that ProxySelector/Authenticator will be used instead. But meh.

Reply:
Fixed (to remove reference on `internal`)